dnstap/dtstream.c

```diff
@@ -1,5 +1,5 @@
 /*
- * dnstap/dtstream.c - Frame Streams implementation for unbound DNSTAP
+ * dnstap/dtstream.h - Frame Streams thread for unbound DNSTAP
```

Comment:
dtstream.h -> dtstream.c

Reply:
Yes, you are right, https://github.com/NLnetLabs/unbound/pull/164/commits/5aa8ae510ed0b69cdb6daf8aca1fe00c1d625d82 fixes it.